src/Microsoft.AspNetCore.Mvc.Razor/Internal/CompilerCacheResult.cs

```diff
@@ -58,6 +60,7 @@ public CompilerCacheResult(IList<IChangeToken> expirationTokens)
             CompilationResult = default(CompilationResult);
             Success = false;
```

Comment:
Can we made `Success` a computed field? Worried about the size of this struct growing too large